libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/lifecycle/UIComponent.kt

```diff
@@ -33,7 +33,7 @@ open class UIComponent : MapboxNavigationObserver {
     }
 
     protected inline fun <T> Flow<T>.observe(crossinline action: suspend (value: T) -> Unit) {
-        coroutineScope.launch { collect(action) }
+        //coroutineScope.launch { collect(action) } //SBNOTE: fixme restore this
```

Comment:
For some reason this was giving a compiler error. Probably in some way related to using the map snapshot. This line should be restored before merging.